app/src/main/java/ch/sdp/vibester/activity/profile/MyProfileActivity.kt

```diff
@@ -53,13 +54,20 @@ class MyProfileActivity : ProfileActivity() {
      */
     private fun setChangeImageBtnListener() {
         findViewById<ImageView>(R.id.avatar).setOnClickListener {
-            showGeneralDialog( "Do you want to change your profile picture?", false)
+            showDialogWhenChangeImage()
         }
         findViewById<CardView>(R.id.myCardView).setOnClickListener {
-            showGeneralDialog( "Do you want to change your profile picture?", false)
+            showDialogWhenChangeImage()
```

Comment:
do you know why there are two clicks for the same functionality?

Reply:
no, I just saw the duplication and refactored them.

Reply:
> no, I just saw the duplication and refactored them.

I will ask in the group